### PR TITLE
[ID] Add `Tournament Committee`

### PR DESCRIPTION
**Pull request by Nivalyx**
Indonesian localization of the newly created `Tournament Committee` wiki page (http://osu.ppy.sh/wiki/People/Tournament_Committee).

---

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Tournament_Committee/id.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Tournament Committee
+
+**Tournament Committee** merupakan sebuah panel yang beranggotakan nama-nama yang telah banyak menyelenggarakan atau berkiprah di dalam ranah turnamen.
```
Comment (ZeroPyrozen) [inline wiki/People/Tournament_Committee/id.md:3]:
```suggestion
**Tournament Committee** merupakan sebuah panel yang beranggotakan pengguna-pengguna yang telah banyak menyelenggarakan atau berkiprah di dalam ranah turnamen.
```

File: wiki/People/Tournament_Committee/id.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Mereka bertanggung jawab untuk:
+
+- Menindaklanjuti laporan-laporan yang diterima dari [formulir pelaporan turnamen](https://pif.ephemeral.ink/tournament-reports)
+- Mengamati berbagai perkembangan yang terjadi di dalam ranah turnamen secara umum dan memberi saran kepada [account support team](/wiki/People/The_Team/Account_support_team) seputar tindakan apa-apa saja yang sebaiknya diambil dalam tiap-tiap kasus pelanggaran yang terjadi
```
Comment (ZeroPyrozen) [inline wiki/People/Tournament_Committee/id.md:8]:
```suggestion
- Mengawasi berbagai perkembangan yang terjadi di dalam ranah turnamen secara umum dan memberi saran kepada [account support team](/wiki/People/The_Team/Account_support_team) seputar tindakan apa-apa saja yang sebaiknya diambil dalam tiap-tiap kasus pelanggaran yang terjadi
```

File: wiki/People/Tournament_Committee/id.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Memperbaharui informasi yang terdapat pada laman [Official tournament support](/wiki/Tournaments/Official_support) secara berkala
```
Comment (ZeroPyrozen) [inline wiki/People/Tournament_Committee/id.md:9]:
```suggestion
- Memperbaharui informasi yang terdapat pada laman wiki [Official tournament support](/wiki/Tournaments/Official_support) secara berkala
```

Comment (ZeroPyrozen) [reply wiki/People/Tournament_Committee/id.md:9]: `laman wiki` or `artikel wiki`

**Review by ZeroPyrozen — COMMENTED** [review]
Just need small change

**Comment by Nivalyx** [comment]
@ZeroPyrozen applied all 👍

**Review by ZeroPyrozen — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**